roles/oc-patch-file-to-configmap/Readme.md

```diff
@@ -0,0 +1,26 @@
+# oc-patch-file-to-configmap-role
+
+The contents of this dir are a copy of the repo at
+https://github.com/aerogear/oc-patch-file-to-configmap-role at commit
+25d961b37d9c3ddc3fadc9544af785bd781c5bbd
```

Comment:
I have put a reference to the commit this came from both here in the Readme, and in the commit message.